easypost/requestor.py

```diff
@@ -0,0 +1,193 @@
+import datetime
+import json
+import platform
+import ssl
+import time
+from urllib.parse import urlencode
+
+from easypost.easypost_object import EasyPostObject
+from easypost.error import Error
+
+
+class Requestor(object):
+    def __init__(self, local_api_key=None):
+        self._api_key = local_api_key
+
+    @classmethod
+    def _objects_to_ids(cls, param):
+        if isinstance(param, EasyPostObject):
```

Comment:
# Reviewer Note

This was the only code change throughout the whole PR: We swapped the isinstance of a `Resource` for `EasyPostObject` which should be safe since Resource inherits from EasyPostObject but doesn't extend it aside from adding functions. Making this change resolves a circular import.